packages/react-native-executorch/common/rnexecutorch/models/speech_to_text/SpeechToText.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "ReactCommon/CallInvoker.h"
+#include "executorch/runtime/core/evalue.h"
+#include <cstdint>
+#include <memory>
+#include <rnexecutorch/models/EncoderDecoderBase.h>
+#include <rnexecutorch/models/speech_to_text/SpeechToTextStrategy.h>
+#include <span>
+#include <vector>
+
+namespace rnexecutorch {
+class SpeechToText : public EncoderDecoderBase {
+public:
+  SpeechToText(const std::string &encoderPath, const std::string &decoderPath,
+               const std::string &modelName,
+               std::shared_ptr<react::CallInvoker> callInvoker);
+  void encode(std::span<float> waveform);
+  int64_t decode(std::vector<int64_t> prevTokens);
+
+private:
+  const std::string modelName;
```

Comment:
There is a clang tidy check for such const in members
```suggestion
  std::string modelName;
```